resolwe/permissions/tests/test_tool.py

```diff
@@ -35,8 +43,8 @@ def test_get_list_admin(self):
         self.assertEqual(len(resp.data), 2)
 
     def test_post(self):
-        resp = self._post({'name': 'New process'}, self.admin)
-        self.assertEqual(resp.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
+        resp = self._post(self.post_data, self.admin)
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
 
     def test_patch(self):
```

Comment:
We need to allow patch and test for HTTP_200_OK here. Development would be a real pain otherwise.